### DISCUSSION TITLE
Mutation Events - Chrome 127+ Support EOL - Migration Caveats

### DISCUSSION THREAD

**Discussion by nrdevau** [mdn/mdn-community#718] [Content]
Hi Peeps,

I've just been bitten by some existing functionality that relied on `DOMSubtreeModified` (a decade since being deprecated I know :cry:) event listening dying in the latest Chrome.

I've been reading through the docs and believe this JSFiddle should show a simple migration for my use case.

There are some caveats though, which may be painpoints to a number of other devs in the next few days as end users browsers get updated.

I think it would be good to document those caveats somewhere.

I believe this is the place that makes the most sense.

Here is a link to a discussion in the jQuery discussions (as I had incorrectly blamed jQuery for this change in behaviour): https://github.com/jquery/jquery/discussions/5535

I doubt many people will find this, and hope even fewer devs are affected by this. In a perfect world all devs would have migrated long ago, but in reality some businesses have pirates that do dirty dev and then it gets so unstable, it ends up landing on the plate of someone like me :joy: 

Anyway, enough ranting, but I just wanted to call it out in case it helps someone else.

Enjoy: https://jsfiddle.net/nrdev/kwLu2yvh/24/

**Comment by nrdevau** [comment]
oh there's also this https://chromestatus.com/feature/5083947249172480 which is what Chrome references when it detects you are listening to a deprecated Mutation Event

**Comment by nrdevau** [comment]
Keyword stuffing
```
DOMSubtreeModified
DOMNodeInserted
DOMNodeRemoved
DOMNodeRemovedFromDocument
DOMNodeInsertedIntoDocument
DOMCharacterDataModified
```

(I have no idea if this will increase visibility or not)

**Comment by pransh15** [comment]
Hi @nrdevau 👋 
Thanks for starting the discussion, and 10/10 on the rant, absolutely love the passion! I reckon the world would appreciate a blog on this for complete caveat documentation in case someone stumbles upon it in the coming weeks.

Also, the JS Fiddle does not seem to be working for me.

**Reply by nrdevau** [reply]
@pransh15 
It works on my machine :sweat_smile: 
Apparently Safari just straight up dies when loading jsfiddle at the moment, it might be related to that?
Sorry, I need to know about 3 more details to provide any more useful diagnostic steps:
1. What browser you're using
2. Do other jsfiddles load correctly?
3. IDK, I can't count ;)
